### DISCUSSION TITLE
Jalali / Persian Calendar Support?

### DISCUSSION THREAD

**Discussion by ParsaGachkar** [Megabit/Blazorise#4118] [Q&A]
any plan to support Jalali / Persian Calendar for Date components?

**Comment by stsrki** [comment]
Considering that we know zero Persian, it would be hard to it for us. But we're always open to contributions. Check one of the recent PRs in case you wish to make it ;)

#4091

**Comment by ParsaGachkar** [comment]
ok good news it work's fine with setting culture, just we need to localaize strings with [https://learn.microsoft.com/en-us/dotnet/api/system.globalization.datetimeformatinfo.monthnames?view=net-6.0](https://learn.microsoft.com/en-us/dotnet/api/system.globalization.datetimeformatinfo.monthnames?view=net-6.0) instead of static strings. at [https://github.com/Megabit/Blazorise/blob/a64b6a027680126d5051a3d46873394d2eb8e1dc/Source/Blazorise/Components/DatePicker/DatePicker.razor.cs#L269](https://github.com/Megabit/Blazorise/blob/a64b6a027680126d5051a3d46873394d2eb8e1dc/Source/Blazorise/Components/DatePicker/DatePicker.razor.cs#L269)
